backend/utils/other/hume.py

```diff
@@ -0,0 +1,176 @@
+import os
+import requests
+
+class HumePredictionEmotionResponseModel:
+    def __init__(
+            self,
+            name: str,
+            score: float,
+    ) -> None:
+        self.name = name
+        self.score = score
+
+    @classmethod
+    def from_dict(cls, data: dict) -> "HumePredictionEmotionResponseModel":
+        model = cls(data["name"], data["score"])
+        return model
+
+
+class HumeJobModelPredictionResponseModel:
+    def __init__(
+            self,
+            emotions=[],
+            emotion_dict={},
+    ) -> None:
+        self.emotions = emotions
+        self.emotions_dict = emotion_dict
+
+    def get_top_emotion_names(self, k: int = 5, peak_threshold:float = .7):
+        n = len(self.emotions_dict)
+        emotions_average = {}
+        for emotion, score in self.emotions_dict.items():
+            if score >= peak_threshold:
+                emotions_average[emotion] = score / n
+
+        ascend_sorted_emotion_average = sorted(emotions_average, key=emotions_average.get, reverse=True)
+        k = min(k, len(ascend_sorted_emotion_average))
+        return ascend_sorted_emotion_average[:k]
+
+    @ classmethod
+    def from_dict(cls, prediction_model: str, data: dict) -> "HumeJobModelPredictionResponseModel":
```

Comment:
Can we filter only by segments that correspond to the user, and trigger the prompt differently?

I mean, to have a other people have said "x" and felt this emotion.

But the main thing is to filter segments where `is_user = True`, that's when we will know if it's theirs